src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -889,7 +891,7 @@ private void InitializeUpdateTopologyTimer()
                 if (_updateTopologyTimer != null)
                     return;
 
-                _updateTopologyTimer = new Timer(UpdateTopologyCallback, null, TimeSpan.FromMinutes(5), TimeSpan.FromMinutes(5));
+                _updateTopologyTimer = new WeakReferencingTimer(UpdateTopologyCallback, this, TimeSpan.FromSeconds(5), TimeSpan.FromSeconds(5));
```

Comment:
The accidental change `FromMinutes()` to `FromSeconds()` has been reverted in https://github.com/ravendb/ravendb/pull/17967